src/plot/series/mark-series.js

```diff
@@ -78,7 +79,11 @@ class MarkSeries extends AbstractSeries {
             onMouseOver: e => this._valueMouseOverHandler(d, e),
             onMouseOut: e => this._valueMouseOutHandler(d, e)
           };
-          return <circle {...attrs} />;
+
+          if (typeof markType === 'function') {
```

Comment:
maybe just check for presence rather than typeof? it seems like the prop-types should type check that?